src/app/components/table_details/table_details.html

```diff
@@ -8,8 +8,10 @@ <h6>Details</h6>
                         <div class="detail-body">
                             <dl class="detail"
                                 ng-repeat="(k, v) in meta">
-                                <dt class="detail-label">{{ k }}</dt>
-                                <dd class="detail-value">{{ v }}</dd>
+                                <div ng-if="k != 'meta_docs'">
```

Comment:
Nice! While there, can you remove the `class=something` below?

Reply:
:see_no_evil: Sloppy